src/MenuButton/index.js

```diff
@@ -88,7 +88,7 @@ const MenuButton = ({
     >
       <AriaButton
         aria-label={label}
-        className="button--reset"
+        className="button--reset nds-menubutton-ariaButton"
```

Comment:
Allows `SplitButton` to select the `button` element rendered by `AriaButton`, so we can match the shape of the focus ring to the visually rendered button.